### PR TITLE
Versions dropdown

### LINKED ISSUE #92: Move versions dropdown from head to story wrapper
Currently the versions dropdown is added to the page via head.html. This should be moved to the story wrapper as we should leave head.html just for user added content.

### PR DESCRIPTION
**Pull request by panosvoudouris**
This creates a version dropdown at the top of the story frame. This is now part of the story wrapper instead of head.html. Resolves #92

### CODE REVIEW

File: src/components/panel/styles.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-button {
+.blabbr-panel-container button {
```
Comment (panosvoudouris) [inline src/components/panel/styles.css:1]: Note: This was leaking the style to the storybook interface.

Comment (iamtmrobinson) [reply src/components/panel/styles.css:1]: Thought this might happen... Thanks.

**Comment by iamtmrobinson** [comment]
Hey @panosvoudouris - when I first load I just see the drop-down, but if I hit a version (I'm assuming they shouldn't work yet as I just get 404) and then hit my back button I see the versions in the header too:
Initial load.
After clicking a version, getting 404 and then going back.

**Comment by panosvoudouris** [comment]
@iamtmrobinson the versions will effectively only work when you have a static build hosted somewhere (see the path it tries to resolve). As such it will always give you a 404 on local dev. The issue with the back button looks like some caching in your browser, mine takes me back to the right thing and what you see looks like an older build (this is what it looked like before doing the dropdown).

**Comment by iamtmrobinson** [comment]
Cheers @panosvoudouris - caching issue fixed by a hard reset etc.

I get a couple React warnings, one can be fixed by initialising `versions` to an empty array rather than `null` in `storyWrapper/index.js`.

The other is just complaining about having unique keys:
```
Each child in an array or iterator should have a unique "key" prop. Check the render method of `Versions`.
    in a (created by Versions)
    in Versions (created by StoryWrapper)
```

**Comment by panosvoudouris** [comment]
Sorted @iamtmrobinson

**Pull request merged**